src/net_processing.h

```diff
@@ -61,6 +61,15 @@ struct Peer {
     /** Whether this peer should be disconnected and marked as discouraged (unless it has the noban permission). */
     bool m_should_discourage GUARDED_BY(m_misbehavior_mutex){false};
 
+    /** Protects block inventory data members */
+    Mutex m_block_inv_mutex;
+    /** List of block ids we still have announce.
```

Comment:
a50dee50fd2aee93b41bfb3b04b24939d0c2f6c1

I understand that comment text has been preserved, but maybe
```suggestion
    /** List of block ids we still have to announce.
```

Reply:
Agree. Will fix.

Reply:
Fixed